src/core/mac/link_raw.cpp

```diff
@@ -224,117 +241,130 @@ void LinkRaw::InvokeReceiveDone(otRadioFrame *aFrame, otError aError)
     }
 }
 
+void LinkRaw::StartTransmit(void)
+{
+#if OPENTHREAD_CONFIG_ENABLE_SOFTWARE_CSMA_BACKOFF
+    if ((mRadioCaps & OT_RADIO_CAPS_CSMA_BACKOFF) == 0 && mTransmitFrame->IsCsmaCaEnabled())
+    {
+        StartCsmaBackoff();
+    }
+    else
+#endif // OPENTHREAD_CONFIG_ENABLE_SOFTWARE_CSMA_BACKOFF
+    {
+        mPendingOperations |= kOperationTransmitData;
+        mOperationTask.Post();
+    }
+}
+
+void LinkRaw::TransmitNow(void)
+{
+    otError error = otPlatRadioTransmit(&GetInstance(), mTransmitFrame);
+
+    if (error != OT_ERROR_NONE)
+    {
+        InvokeTransmitDone(mTransmitFrame, NULL, error);
```

Comment:
How about using `OT_ERROR_ABORT` in TransmitDone?

Reply:
I will update the code.